graylog2-web-interface/src/components/indices/IndicesConfiguration.jsx

```diff
@@ -22,21 +22,12 @@ import { Row, Col } from 'components/graylog';
 import IndexMaintenanceStrategiesSummary from 'components/indices/IndexMaintenanceStrategiesSummary';
 import 'components/indices/rotation'; // Load rotation plugin UI plugins from core.
 import 'components/indices/retention'; // Load rotation plugin UI plugins from core.
-import style from 'components/configurations/ConfigurationStyles.lazy.css';
```

Comment:
I looked through our rotation and retention plugins and could not find any of these styles being used in there, so I think we don't need the file anymore.